lib/studies/reply/app.dart

```diff
@@ -15,14 +15,26 @@ class ReplyApp extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final galleryTheme = GalleryOptions.of(context).themeMode;
+    bool isDark;
+
+    if ((galleryTheme == ThemeMode.dark) | (galleryTheme == ThemeMode.light)) {
+      isDark = GalleryOptions.of(context).themeMode == ThemeMode.dark;
+    } else {
+      isDark = MediaQuery.of(context).platformBrightness == Brightness.dark;
+    }
+    final replyTheme =
+        isDark ? _buildReplyDarkTheme(context) : _buildReplyLightTheme(context);
+
     return MultiProvider(
       providers: [
         ChangeNotifierProvider<EmailStore>.value(value: EmailStore()),
       ],
       child: MaterialApp(
         title: 'Reply',
         debugShowCheckedModeBanner: false,
-        theme: _buildReplyLightTheme(context),
+        darkTheme: replyTheme,
```

Comment:
Instead of having to check `isDark` everywhere, is it possible to utilize these properties and the colorScheme instead?

Then throughout the app you would just read the value from the `Theme.of(context).colorScheme`?